lib/hyacinth/adapters/preservation_adapter/fedora3/dc_properties.rb

```diff
@@ -0,0 +1,79 @@
+module Hyacinth
+  module Adapters
+    module PreservationAdapter
+      class Fedora3::DCProperties
+        XML_PREFIX = '<oai_dc:dc xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/"'\
+                     ' xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">'.freeze
+        XML_SUFFIX = '</oai_dc:dc>'.freeze
+
+        include Fedora3::PidHelpers
+        include Fedora3::TitleHelpers
+        include Fedora3::DatastreamMethods
+
+        def self.from(hyacinth_obj)
+          new(hyacinth_obj)
+        end
+
+        def initialize(hyacinth_obj)
+          @hyacinth_obj = hyacinth_obj
+        end
+
+        def to(fedora_obj)
+          proposed_dc_properties = proposed_dc_properties(@hyacinth_obj)
+          current_dc_properties = parse_dc_xml(fedora_obj)
+          fedora_obj.datastreams['DC'].content = to_dc_xml(proposed_dc_properties) unless current_dc_properties.eql?(proposed_dc_properties)
+        end
+
+        def proposed_dc_properties(hyacinth_obj = @hyacinth_obj)
+          properties = {}
+          # set the title
+          properties[:title] = [get_title(hyacinth_obj.dynamic_field_data)]
+          # set the identifiers
+          properties[:identifier] = hyacinth_obj.identifiers&.to_a || []
+          # add the pid as an identifier
+          properties[:identifier].concat Array.wrap(digital_object_pids(hyacinth_obj))
+          if hyacinth_obj.digital_object_type.casecmp('asset').zero?
```

Comment:
Are you concerned here that when digital_object_type returns "asset" here, that "asset" may not always be all lower case? (thus the use of `casecmp('asset').zero?`)

Reply:
Me a month ago is a different person, really - if you think it's unclear or unnecessary, we can just delete it.

Reply:
I believe you!  Deleting sounds good, thanks.